### DISCUSSION TITLE
Ability to customize the order of tables in Explore view

### DISCUSSION THREAD

**Discussion by bentqc** [grafana/grafana#58674] [Explore]
Is there any plans for implementing custom ordering of the explore tables such as inverting the list from older to newer entries rather than newer to older entries ? 

right now we would like to list traces from oldest to newest starting from the top but in order to do this, you have to click the column name. 

Perhaps there could be extra options to allow this from the query details (where you can select servicename, tags, limits).

thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
